CMakeLists.txt

```diff
@@ -136,6 +136,7 @@ add_executable(gmapping src/slams/gmapping/gmapping.cpp)
 add_executable(tiny_slam src/slams/tiny/tiny_slam.cpp)
 add_executable(viny_slam src/slams/viny/viny_slam.cpp)
 add_executable(viny_slam_x src/slams/vinyx/vinyx_slam.cpp)
+add_executable(path_publisher src/utils/path_publisher.cpp)
```

Comment:
`path_publisher.cpp` should be moved from `src/utils` to `src/ros`. The `utils` folder contains classes related to the framework itself and `ros` - ROS adapters for the framework.